src/main/java/donnafin/logic/parser/ParserUtil.java

```diff
@@ -240,25 +240,30 @@ public static Money parseMoney(String money) throws ParseException {
         String trimmedCommission = money.trim();
```

Comment:
We should give this variable a more appropriate name :). Not sure where commission comes in

Reply:
rekt. excellent point!